frontend/src/features/admin-form/settings/mutations.ts

```diff
@@ -131,7 +131,7 @@ export const useMutateFormSettings = () => {
         const toastStatusPublicMessage =
           newData.responseMode === FormResponseMode.Encrypt
             ? `Your form is now open.\n\nStore your secret key in a safe place. If you lose your secret key, all your responses will be lost permanently.`
-            : `Your form is now open.\n\nIf you expect a large number of responses,  [AutoArchive your mailbox](${GUIDE_PREVENT_EMAIL_BOUNCE}) to avoid losing any of them.`
+            : 'Your form is now open.'
```

Comment:
Thanks for contributing this suggestion! Quick thought:

- if the FormResponseMode is Encrypt (Storage): no change to the text is expected
- if the FormResponseMode is Multirespondent (MRF): we still use a secret key, so we should probably be displaying the same message used for Storage mode!
- and we've removed FormResponseMode.Email as an option in practice, so that's sorted.

```suggestion
        const toastStatusPublicMessage =
          `Your form is now open.\n\nStore your secret key in a safe place. If you lose your secret key, all your responses will be lost permanently.`
```